src/views/WaterTrackingView.jsx

```diff
@@ -4,36 +4,33 @@ import Title from "antd/es/typography/Title";
 const { Header, Content } = Layout;
 const { useMessage } = message;
 
-const WATER_GOAL = 100;
+const WATER_GOAL = 8;
 
 const createWaterChangeMessage = (messageInstance, value) => {
   if (value < 0) {
-    messageInstance.success(`Removed ${value} glass of water`);
+    messageInstance.error(`Removed ${value} glass of water`);
     return;
   }
 
-  messageInstance.error(`Added ${value} glass of water`);
+  messageInstance.success(`Added ${value} glass of water`);
 };
 
 const getProgressColor = (currentIntake, goal) => {
-  return "#50C878";
 
-  if (currentIntake < (2 / 100) * goal) {
+  if (currentIntake < (25 / 100) * goal) {
+    return "#FF5050 ";
+  } else if (currentIntake < (50 / 100) * goal) {
```

Comment:
Here each condition block returns, so the function ends there. In this situation we can skip using else if and just use the next if block

```js
if(...){
   return ...
}

if(...){
   return ...
}

return ...
```